source/building.rst

```diff
@@ -19,6 +19,14 @@ Sphinx 构建网站
 
     若未安装 Python，建议通过 :doc:`Anaconda <software:anaconda/index>`  来安装和管理 Python。
 
+.. dropdown:: :fa:`exclamation-circle,mr-1` 安装 Sphinx
+   :container: + shadow
+   :title: bg-info text-white font-weight-bold
+
+   可以使用 Python 自带的工具 pip 来安装 Sphinx::
+   
+       $ pip install sphinx
+
```

Comment:
Perhaps we should remove it, since sphinx is already listed in `requirements.txt`.

Reply:
Done in 7ccbad2.